apps/wallet/src/ui/app/shared/social-buttons/SocialButtons.tsx

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { SocialTwitch24, SocialMicrosoft24, SocialGoogle24, SocialFacebook24 } from '@mysten/icons';
+import { type ComponentProps } from 'react';
+import { BaseSocialButton } from './BaseSocialButton';
+
+export type SignInWithProviderButtonProps = {
+	showLabel?: boolean;
+} & Omit<ComponentProps<typeof BaseSocialButton>, 'variant' | 'icon' | 'label'>;
+
+export function SignInWithGoogleButton(props: SignInWithProviderButtonProps) {
+	return <BaseSocialButton label="Sign in with Google" icon={SocialGoogle24} {...props} />;
+}
+
+export function SignInWithMicrosoftButton(props: SignInWithProviderButtonProps) {
+	return <BaseSocialButton label="Sign in with Microsoft" icon={SocialMicrosoft24} {...props} />;
+}
+
+export function SignInWithFacebookButton(props: SignInWithProviderButtonProps) {
```

Comment:
I'm not super opposed to these exports existing but I also could see you just keeping the base button, and then defining these props on usage vs having separate component versions of them all. Could see either way working out